docs/management/admin/response-actions-config.asciidoc

```diff
@@ -25,66 +24,107 @@ preview::[]
+.**Set up CrowdStrike response actions**
+[%collapsible]
+====
+// NOTE TO CONTRIBUTORS: These sections have very similar content. If you change anything 
+// in this section, apply the change to the other sections, too.
+
+. **Create an API client in CrowdStrike.** Refer to CrowdStrike's docs for instructions on creating an API client.
++
+- Give the API client the minimum privilege required to read CrowdStrike data and perform actions on enrolled hosts.
+- Take note of the client ID, client secret, and base URL; you'll need them in later steps when you configure {elastic-sec} components to access CrowdStrike.
+
+. **Install the CrowdStrike integration and {agent}.** Elastic's {integrations-docs}/crowdstrike[CrowdStrike integration]
+ collects and ingests logs into {elastic-sec}.
++
+.. Go to **Integrations**, search for and select **CrowdStrike**, then select **Add CrowdStrike**.
+.. Configure the integration with an **Integration name** and optional **Description**.
+.. Select **Collect CrowdStrike logs via API**, and enter the required **Settings**:
+   - **Client ID**: Client ID for your CrowdStrike API client.
+   - **Client Secret**: Client secret allowing you access to CrowdStrike.
+   - **URL**: The base URL of the CrowdStrike API.
+.. Select the **Falcon Alerts** and **Hosts** sub-options under **Collect CrowdStrike logs via API**.
+.. Scroll down and enter a name for the agent policy in **New agent policy name**. If other agent policies already exist, you can click the **Existing hosts** tab and select an existing policy instead. For more details on {agent} configuration settings, refer to {fleet-guide}/agent-policy.html[{agent} policies].
+.. Click **Save and continue**.
+.. Select **Add {agent} to your hosts** and continue with the <<enroll-agent,{agent} installation steps>> to install {agent} on a resource in your network (such as a server or VM). {agent} will act as a bridge collecting data from CrowdStrike and sending it back to {elastic-sec}.
+
+. **Create a CrowdStrike connector.** Elastic's {kibana-ref}/action-types.html[CrowdStrike connector] enables {elastic-sec} to perform actions on CrowdStrike-enrolled hosts.
++
+IMPORTANT: Do not create more than one CrowdStrike connector.
++
+.. Go to **Stack Management** → **Connectors**, then select **Create connector**.
+.. Select the CrowdStrike connector.
+.. Enter the configuration information:
+   - **Connector name**: A name to identify the connector.
+   - **CrowdStrike API URL**: The base URL of the CrowdStrike API.
+   - **CrowdStrike Client ID**: Client ID for your CrowdStrike API client.
+   - **Client Secret**: Client secret allowing you access to CrowdStrike.
+.. Click **Save**.
+
+. **Create and enable a rule to generate {elastic-sec} alerts.** (Optional) Create a <<create-custom-rule,custom query detection rule>> to generate {elastic-sec} alerts based on CrowdStrike events and data. Use the index pattern `logs-crowdstrike*`.
```

Comment:
This, IMO, is not a good suggestion for the index name - it essentially says "create alerts for any type of crowdstrike data". I think (but you would need to confirm with @tomsonpl ) that there are specific indicates for crowdstrike alerts and other malware data, so perhaps using a more specific example would be best.

The other alternative would be to refer the customer tot he Integration overview page which describes the different types of data collected by the integration - that should give the user a good starting point on how to setup their Rule to only promote valid threats to SIEM alerts in kibana.

____

@caitlinbetz  - it would be great if we could get an example rule from one of our customers that run CrowdStrike just to see how they have setup their rule

Reply:
A sample index pattern or query would be preferable, I think — linking to [the integration's docs](https://www.elastic.co/docs/current/en/integrations/crowdstrike) could be overwhelming since it just lists every possible fields ingested, but doesn't recommend any particular pattern.

Is there an alert-specific pattern we could suggest, similar to how for SentinelOne we suggest `logs-sentinel_one.alert*`? That way the query only targets alerts, not any type of CrowdStrike data.

Reply:
I would agree with @paul-tavares here, it's really up to user what query they want to use and most of the times they are pretty specific. For the docs I would suggest using something like this: 'use a query or index pattern that could find Crowdstrike data'   - of course in better English ;p 

WDYT @joepeeples ?

Reply:
Similar to above, I'll reword this to be more general guidance, and probably link to our CrowdStrike integration docs page since it lists all the available fields. Even if it's up to the user to write queries however they see fit, it'd be ideal if we could at least give some example queries to illustrate how to detect specific things, but maybe we can add that in a later revision once we know more about the data.